vms/registration/views.py

```diff
@@ -90,7 +90,8 @@ def post(self, request):
                         })
                 try:
                     admin_country_name = request.POST.get('country')
-                    admin_country = Country.objects.get(name=admin_country_name)
+                    admin_country = Country.objects.get(
+                        name=admin_country_name)
```

Comment:
Here, nothing is changed.!

Reply:
Flake8 error was getting displayed ,words limit per line.

Reply:
@decon-harsh I don't think that the line is actually too long, you should revert it. flake8 shows this for line length greater than 88, but you should do it if there are more that one comma seaprated values here